File: android/app/src/main/java/enzio/android/eatfit/data/model/AuthModels.kt
package enzio.android.eatfit.data.model

data class LoginRequest(
    val loginIdentifier: String,
    val password: String
)

data class RegisterRequest(
    val email: String,
    val login: String,
    val password: String,
    val passwordConfirmation: String
)

data class AuthResult(
    val success: Boolean,
    val message: String = "",
    val accessToken: String = "",
    val refreshToken: String = "",
    val userId: String = "",
    val email: String = "",
    val login: String = "",
    val resetToken: String = ""
)

File: desktop/EatfitDesktop/Models/AuthModels.cs
namespace EatfitDesktop.Models
{
    public class LoginModel
    {
        public string LoginIdentifier { get; set; } = string.Empty;
        public string Password { get; set; } = string.Empty;
    }

    public class RegisterModel
    {
        public string Email { get; set; } = string.Empty;
        public string Login { get; set; } = string.Empty;
        public string Password { get; set; } = string.Empty;
        public string PasswordConfirmation { get; set; } = string.Empty;
    }

    public class AuthResult
    {
        public bool Success { get; set; }
        public string Message { get; set; } = string.Empty;
        public string AccessToken { get; set; } = string.Empty;
        public string RefreshToken { get; set; } = string.Empty;
        public string UserId { get; set; } = string.Empty;
        public string Email { get; set; } = string.Empty;
        public string Login { get; set; } = string.Empty;
    }

    public class ResetPasswordModel
    {
        public string LoginIdentifier { get; set; } = string.Empty;
        public string VerificationCode { get; set; } = string.Empty;
        public string NewPassword { get; set; } = string.Empty;
        public string PasswordConfirmation { get; set; } = string.Empty;
    }
}